content/docs/releases/README.md

```diff
@@ -16,21 +16,18 @@ and other world events.
 <a id="supported-releases"></a>
 ## Currently supported releases
 
-| Release  | Release Date |      End of Life       | [Supported Kubernetes versions][s] | [Supported OpenShift versions][s] |
-|----------|:------------:|:----------------------:|:----------------------------------:|:---------------------------------:|
-| [1.13][] | Sep 12, 2023 |    Release of 1.15     |            1.23 → 1.28             |           4.10 → 4.15             |
-| [1.12][] | May 19, 2023 |    Release of 1.14*    |            1.22 → 1.27             |            4.9 → 4.14             |
+|   Release    | Release Date |      End of Life       | [Supported Kubernetes versions][s] | [Supported OpenShift versions][s] |
+|--------------|:------------:|:----------------------:|:----------------------------------:|:---------------------------------:|
+|     [1.13][] | Sep 12, 2023 |    Release of 1.15     |            1.23 → 1.28             |           4.10 → 4.15             |
+| [1.12 LTS][] | May 19, 2023 |      May 19, 2025      |            1.22 → 1.28             |            4.9 → 4.15             |
```

Comment:
self review: I think we'll probably update these to also support k8s 1.29 but since we aren't testing that yet I've held back on updating these.